docs/src/main/asciidoc/hibernate-orm-guide.adoc

```diff
@@ -267,6 +276,53 @@ same project where you build your Quarkus application, everything will work fine
 or jars, you can make sure that your jar is treated like a Quarkus application library by adding an empty `META-INF/beans.xml` file.
 This will allow Quarkus to index and enhance your entities as if they were inside the current project.
 
+== Hibernate ORM in development mode
+
+Quarkus development mode is really useful for applications that mix front end or services and database access.
+There are a few common approaches.
+
+The first choice is to use `quarkus.hibernate-orm.database.generation=drop-and-create` in conjunction with `import.sql`.
+That way for every change to your app and in particular to your entities, the database schema will be properly recreated
+and your data fixture (stored in `import.sql`) will be used to repopulate it from scratch.
```

Comment:
Although not technically improper here, your usage of the term "fixutre" may expose us to the first example mentioned here: https://en.wikipedia.org/wiki/Test_fixture#Software "The Ruby on Rails web framework uses YAML to initialize a database before running a test", which is what I'm used to in Play too. It's an orthogonal issue, but I guess I should open an issue to discuss it.

Reply:
ok, if there is a better name, I'm happy to use it. Technically import.sql is used to init the DB before running a test.